02_sequencing/README.md

```diff
@@ -20,5 +20,13 @@ This column is optional. If undefined will default to 1/30th of a second.
 Examples are `R_0`, `G_0` and `B_0` which are the red, green and blue channel for LED 0.
 The values of these columns should be floats or ints between 0 and 255 inclusive. 
 
-⚠️Note that the old python running code has a number of limitations.
+### Warning ⚠️
+The old running code has a number of limitations including assuming the position of columns.
+
+To guarantee that your spreadsheet is read correctly the first column should be `FRAME_ID` or `FRAME_TIME`.
```

Comment:
that doesn't match with the top of the README now, there it explicitly states FRAME_ID *then* FRAME_TIME

Reply:
are we trying to be backwards-compatible again?
because in one place the old code has been mentioned as being discarded

Reply:
The part at the top doesn't mention order at all. In theory the columns could be in any order and it should run just fine. The new code does support that but for compatibility with the old code it should be formatted like that.

The old code does not allow columns to be added so the only options we have are to either replace the unused FRAME_ID column or break the old code.
Discussion for this can be found here and the few comments below that.
https://github.com/GSD6338/XmasTree/issues/2#issuecomment-1001513239

Reply:
okay then, changed format for my visualiser to that one, FRAME_ID isn't useful anyway, i might write a converter

Reply:
Note that I have changed FRAME_TIME to miliseconds as others suggested